.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,66 @@
+name: Code Coverage
+on:
+  pull_request:
+    branches-ignore: [dependabot/**]
+jobs:
+  comment-coverage-difference:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v2 # TODO: does this node setup need to happen each time we switch branches?
+        with:
+          node-version: 12.x
+          cache: yarn
+      - name: yarn install for this PR
+        run: yarn install # TODO: do we need --frozen-lockfile? we are yarn install-ing again below
```

Comment:
`--frozen-lockfile` won't hurt but it's not needed since the Drone CI will already cover that check.